scripts/jenkins/ardana/ansible/run-crowbar-tests.yml

```diff
@@ -31,15 +31,15 @@
             state: present
           loop:
             - "python-setuptools"
-            - "python-os-testr" # pulls python-subunit and includes subunit2html.py
+            - "python-os-testr" # pulls python-python-subunit and includes subunit2html.py
+            - "python-virtualenv"
 
         - name: Ensure subunit tools venv exists
           pip:
             name: "{{ item }}"
             virtualenv: "{{ subunit_tools_venv }}"
           loop:
             - "junitxml"
-            - "python-subunit"
```

Comment:
Some of the invocations below might rely on it being installed in the venv location, e.g. perhaps `{{ subunit_tools_venv }}/bin/subunit-stats`?

Reply:
Totally right. Moreover, the python-python-subunit package doesn't provide subunit-stats. So, adding back subunit to be installed by pip into the virtualenv.